src/main/java/org/candlepin/model/FilterBuilder.java

```diff
@@ -14,52 +14,28 @@
  */
 package org.candlepin.model;
 
-import org.hibernate.Criteria;
-import org.hibernate.criterion.Conjunction;
-import org.hibernate.criterion.Criterion;
-import org.hibernate.criterion.DetachedCriteria;
-import org.hibernate.criterion.LikeExpression;
-import org.hibernate.criterion.Restrictions;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
 import java.util.Collection;
 import java.util.Collections;
 import java.util.HashMap;
 import java.util.LinkedList;
 import java.util.List;
 import java.util.Map;
-import java.util.Map.Entry;
-import java.util.regex.Matcher;
-import java.util.regex.Pattern;
-
 
 
 /**
- * FilterBuilder
- *
  * Contains the logic to apply filter Criterion to a base criteria.
  */
 public abstract class FilterBuilder {
-    private static final Logger log = LoggerFactory.getLogger(FilterBuilder.class);
-
-    public static final String WILDCARD_REGEX = "((?:[^*?\\\\]*(?:\\\\.?)*)*)([*?]|\\z)";
-    public static final Pattern WILDCARD_PATTERN = Pattern.compile(WILDCARD_REGEX);
```

Comment:
No. It was there but it was all unused.

Reply:
That's a bit concerning. Did we drop wildcard support for the lookups these queries back?

Reply:
The pattern was still used by the FilterLikeExpression, which was used by FactFilterBuilder. There was also at one point some overlap with the subman --matches filtering stuff which needs to be retained.

Reply:
So it looks like this PR doesn't make any actual changes, but just removes some old dead code. The bit of code in question that handles the matches filtering for the various endpoints has long since been moved to other bits of logic (`.sanitizeMatchesFilter` and `.buildConsumerFactPredicate`), so the stuff from FilterBuilder was entirely unused.

The title of the PR and commit imply a lot more than what is actually occurring.